### ISSUE TITLE
gc_manager error format does not meet expectations

### ISSUE THREAD

**Issue by superlzs0476** [tikv/tikv#8294]
### What version of TiKV are you using?

v4.0.2
### Steps to reproduce

Because of the region has been scheduled, the GC delete region happened error and try again later. 

### What did you expect?
```log
[2020/07/16 15:12:46.102 +08:00] [WARNING] [gc_manager.rs:533] ["failed gc"] [err="Engine(Request(message: \"stale command\"))"] [end_key=748000000000003EFFB75F698000000000FF00000D0003800000FF0000015CAB000000FC] [region_epoch="Some(conf_ver: 6761 version: 11644)"] [region_id=15161048] offset:307,243,524 source:/data/tikv/tikvlog/tikv.logtype:log _id:AXNWeAkcWV1P0-Qo9HjW _type:doc _index:logstash-tikv-2020.07.16 _score: -
```
### What did happened?
```log
[2020/07/16 15:12:46.102 +08:00] [ERROR] [gc_manager.rs:533] ["failed gc"] [err="Engine(Request(message: \"stale command\"))"] [end_key=748000000000003EFFB75F698000000000FF00000D0003800000FF0000015CAB000000FC] [region_epoch="Some(conf_ver: 6761 version: 11644)"] [region_id=15161048] offset:307,243,524 source:/data/tikv/tikvlog/tikv.logtype:log _id:AXNWeAkcWV1P0-Qo9HjW _type:doc _index:logstash-tikv-2020.07.16 _score: -
```

**Comment by breeswish** [comment]
@MyonKeminta PTAL

**Comment by MyonKeminta** [comment]
The fixing PR is submitted: https://github.com/tikv/tikv/pull/8396

**Comment by MyonKeminta** [comment]
This is expected to be fixed in 3.0.18 and 4.0.6.
